diff -ur ffingerd-1.21.old/Makefile.in ffingerd-1.21/Makefile.in
--- ffingerd-1.21.old/Makefile.in	Wed May 12 23:27:15 1999
+++ ffingerd-1.21/Makefile.in	Thu Jun 24 08:26:35 1999
@@ -21,19 +21,19 @@
 	@echo Ok, do a 'make install' or 'make install.all' now.
 
 install: ffingerd
-	$(INSTALL) -d $(INSTALL_DIR) $(MANDIR) $(MANDIR)/man8
-	$(INSTALL) ffingerd $(INSTALL_DIR)/ffingerd
-	$(INSTALL_DATA) -m 444 ffingerd.8 $(MANDIR)/man8
+	$(INSTALL) -d $(DESTDIR)$(INSTALL_DIR) $(DESTDIR)$(MANDIR) $(DESTDIR)$(MANDIR)/man8
+	$(INSTALL) ffingerd $(DESTDIR)$(INSTALL_DIR)/ffingerd
+	$(INSTALL_DATA) -m 444 ffingerd.8 $(DESTDIR)$(MANDIR)/man8
 	@echo Do not forget to fix /etc/inetd.conf and kill -HUP the inetd.
 
 install.all: ffingerd
-	$(INSTALL) -d $(INSTALL_DIR) $(MANDIR) $(MANDIR)/man8
-	$(INSTALL) ffingerd $(INSTALL_DIR)/ffingerd
-	$(INSTALL_DATA) -m 444 ffingerd.8 $(MANDIR)/man8
-	$(SRC)/install.conf $(INSTALL_DIR)/ffingerd $(AWK) $(SED) $(KILLALL)
+	$(INSTALL) -d $(DESTDIR)$(INSTALL_DIR) $(DESTDIR)$(MANDIR) $(DESTDIR)$(MANDIR)/man8
+	$(INSTALL) ffingerd $(DESTDIR)$(INSTALL_DIR)/ffingerd
+	$(INSTALL_DATA) -m 444 ffingerd.8 $(DESTDIR)$(MANDIR)/man8
+	$(SRC)/install.conf $(DESTDIR)$(INSTALL_DIR)/ffingerd $(AWK) $(SED) $(KILLALL)
 
 uninstall:
-	rm -f $(INSTALL_DIR)/ffingerd $(MANDIR)/man8/ffingerd*
+	rm -f $(DESTDIR)$(INSTALL_DIR)/ffingerd $(DESTDIR)$(MANDIR)/man8/ffingerd*
 	@echo Do not forget to tell inetd...
 
 clean:
