diff -Nru ffingerd-1.26/Makefile.in ffingerd-1.26.new/Makefile.in
--- ffingerd-1.26/Makefile.in	Mon Jan 24 18:08:54 2000
+++ ffingerd-1.26.new/Makefile.in	Mon Jan 31 21:49:48 2000
@@ -26,15 +26,15 @@
 	@echo Ok, do a 'make install' or 'make install.all' now.
 
 install.bin: ffingerd
-	$(SHELL) $(srcdir)/mkinstalldirs $(SBINDIR) $(MANDIR) $(MANDIR)/man8
-	$(INSTALL) ffingerd $(SBINDIR)/ffingerd
-	$(INSTALL) -m 444 $(srcdir)/ffingerd.8 $(MANDIR)/man8
+	$(SHELL) $(srcdir)/mkinstalldirs $(DESTDIR)$(SBINDIR) $(DESTDIR)$(MANDIR) $(DESTDIR)$(MANDIR)/man8
+	$(INSTALL) ffingerd $(DESTDIR)$(SBINDIR)/ffingerd
+	$(INSTALL) -m 444 $(srcdir)/ffingerd.8 $(DESTDIR)$(MANDIR)/man8
 
 install: install.bin
 	@echo Do not forget to fix /etc/inetd.conf and kill -HUP the inetd.
 
 install.all: install.bin
-	$(SRC)/install.conf $(SBINDIR)/ffingerd $(AWK) $(SED) $(KILLALL)
+	$(SRC)/install.conf $(DESTDIR)$(SBINDIR)/ffingerd $(AWK) $(SED) $(KILLALL)
 
 uninstall:
 	rm -f $(SBINDIR)/ffingerd $(MANDIR)/man8/ffingerd*
